boards/mulle/doc.txt

```diff
@@ -3,13 +3,16 @@
 @ingroup     boards
 @brief       Support for Eistec Mulle IoT boards
 
-![Mulle](http://eistec.github.io/images/mulle-small.jpg)
+![Mulle]
+https://web.archive.org/web/20161213064400im_/http://eistec.github.io/images/mulle-small.jpg)
 
 The Mulle is a miniature wireless Embedded Internet System suitable for
 wireless sensors connected to the Internet of Things, and designed for rapid
-prototyping. It can be bought directly from [Eistec AB](http://www.eistec.se).
+prototyping. It can be bought directly from [Eistec AB]
+(https://web.archive.org/web/20190801102538/http://www.eistec.se:80).
```

Comment:
I guess that if the link is not longer valid it one can't buy it anymore, right?

Reply:
That's right. But the board is still supported.

Reply:
Yes sure, but I that there is no point on having this sentence here anymore, it can't be bought, and the link to the archive of the website is already on the list of links.

Reply:
Sentence removed.